File: nan21.dnet.module.hr/nan21.dnet.module.hr.ui.extjs/src/main/webapp/src/ds/CourseTypeLovDs.js
 
   	 
Ext.define("net.nan21.dnet.module.hr.training.ds.model.CourseTypeLovDs" , {
	extend: 'Ext.data.Model',
	fields: [  		
		{name:"id",type:"int", useNull:true }, 		
		{name:"clientId",type:"int", useNull:true }, 		
		{name:"name",type:"string" }, 		
		{name:"active",type:"boolean" }
	]
});
	
Ext.define("net.nan21.dnet.module.hr.training.ds.param.CourseTypeLovDsParam", {
	extend: 'Ext.data.Model',
	fields: [ 
	]
}); 
 

File: nan21.dnet.module.hr/nan21.dnet.module.hr.presenter/src/main/java/net/nan21/dnet/module/hr/training/ds/model/CourseTypeLovDs.java
/*
 * DNet eBusiness Suite
 * Copyright: 2010 Nan21 Electronics SRL. All rights reserved.
 * Use is subject to license terms.
 */
package net.nan21.dnet.module.hr.training.ds.model;

import net.nan21.dnet.core.api.annotation.SortField;
import net.nan21.dnet.core.presenter.model.base.AbstractTypeLov;

import net.nan21.dnet.module.hr.training.domain.entity.CourseType;
import net.nan21.dnet.core.api.annotation.Ds;
import net.nan21.dnet.core.api.annotation.DsField;

@Ds(entity = CourseType.class, jpqlWhere = " e.active = true ", sort = { @SortField(field = CourseTypeLovDs.fNAME) })
public class CourseTypeLovDs extends AbstractTypeLov<CourseType> {

    public CourseTypeLovDs() {
        super();
    }

    public CourseTypeLovDs(CourseType e) {
        super(e);
    }

}
